snap/snapcraft.yaml

```diff
@@ -292,6 +292,8 @@ parts:
     # https://github.com/docker/compose/releases
     source-commit: v2.5.0
```

Comment:
Wouldn't it be better to update this to [v2.16.0](https://github.com/docker/compose/releases/tag/v2.16.0) instead of cherry-picking individual Go dep changes?

Reply:
i'm doing a minimal change update for now. And fighting packaging issues in the snapcraft.yaml as well.

Reply:
And I don't have the confidence to bump to new major version, of such a large jump. Someone else is looking into it after me.